src/app/shared/directives/go-back/go-back.directive.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { GoBackDirective } from "./go-back.directive";
+
+describe("GoBackDirective", () => {
+    it("should create an instance", () => {
+        // const directive: GoBackDirective = new GoBackDirective();
```

Comment:
please remove commented code or fix the test

Reply:
removed